docs/source/learn/core_notebooks/pymc_aesara.ipynb

```diff
@@ -0,0 +1,1893 @@
+{
```

Comment:
<p>There should be a myst target at the beginning of the file: <code>(pymc_aesara)=</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Thanks! I added it in https://github.com/pymc-devs/pymc/pull/5721/commits/54841a3fd0c1b57714d0df6504fcc4d342220933